home.html

```diff
@@ -0,0 +1,164 @@
+                Program
+            </li></a>
+            <a href="#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="#sponsor">
+           <a href="#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        </ul>
+    </div>
+<header class="header flex">
+    <h1 class="mainTitle">
+        "Mellon!"
+    </h1>
+    <h2 class="secondTitle">
+        ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+    </h2>
+</header>
+<main>
+    <div class="index flex">
+        <p class="description">
+            As a Tolkien reader or  a Lord of the Rings watcher, we invite you a fantastic journey in Istanbul with different people all over the world.   
+        </p>
+        <p class="date">
+            2024.03.01 (WED) - 05 (FRI)</p>
+        <p class="location">
+@Taksim Square of İstanbul in Turkey
+        </p>
+    </div>
+    <div class="program flex" id="program">
+        <h2 class="mainProgram">
+            Main Program
+        </h2>
+        <hr class="mainHr"/>
+        <ul class="schedule flex">
+            <li class="scheduleItems flex">
+                <img src="lecture-logo.png" class="scheduleItemImage">
+                <h3 class="scheduleTitle"> 
+                    Opening
+                 </h3> 
+                 <p class="scheduleIndex">
+                    As speaker grandson of J.R.R Tolkien Simon Tolkien
+                 </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="exhibition-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Video Shows
+            </h3>
+            <p class="scheduleIndex">
+                The highlights of series
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="workshop-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle">
+                
+                Workshops
+            </h2>
+            <p class="scheduleIndex">
+                Role playing games, Creating Middle-Earth musics
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="network-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Consers
+            </h3>
+            <p class="scheduleIndex">
+                Sting, Joan Ibanez, Mahmut Orhan, Michael Jackson-Luciano Pavarotti, Nil Karaibrahimgil, Amy Winehouse
+            </p>
+        </li>
+        </ul>
+        <button class="joinButton" id="join">
+            Join!
+        </button>
+<a href="#program"> <p class="wholeProgram">SEE THE WHOLE PROGRAM</p></a>
+    </div>
+   
+    <div class="featuredSpeakers flex">
+<h2 class="featuredTitle">
+    Featured Speakers
+    <hr class="hrFeaturedTitle"/>
+</h2>
```

Comment:
- [ ] The margins/ paddings for this section are not well balanced and the section almost overlaps with the other sections. Kindy adjust the margins/ padding so that the section matches the Behance design.
- [ ] Kindly center this section title so that it matches the design provided in Behance. 
- [ ] Also, the underneath horizontal line is not well aligned and it is distorting the design. Kindly update it as well.

**This is how I am seeing your design:**

![image](https://user-images.githubusercontent.com/86318284/212126440-8bb0d0ae-d3bd-4e70-958e-5a4d9106d2b6.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212126488-cafb6434-6de4-4ad8-a514-12bafd1ada40.png)